docs_src/references/exporter-stdout.md

```diff
@@ -6,19 +6,37 @@ You can launch the stdout exporter this way (running the default powercap_rapl s
 
 	scaphandre stdout
 
-Default behavior is to measure and show metrics periodically during 10 seconds. You can change that timeout with `-t`. Here is how to display metrics during one minute:
+Default behavior is to measure and show metrics periodically during 10 seconds. You can change that timeout with `-t`.
+A value of `-t 0` will display top consumers infinitely and must be interrupted with ctrl-c.
+
+Here is how to display metrics during one minute:
 
     scaphandre stdout -t 60
 
-You can change as well the step measure duration with -s. Here is how to display metrics during one minutes with a 5s step:
+You can change as well the step measure duration with `-s`. Here is how to display metrics during one minutes with a 5s step:
 
     scaphandre stdout -t 60 -s 5
 
+You can change the number of top consumers displayed with `-p`. Here is how to display the first 20 top consumers:
+
+    scaphandre stdout -p 20
+
+You can filter the processes to display with `-r`. A warning will be risen if this option is used with `-p` at the same time.
+In such case, `-p` behavior is disabled.
```

Comment:
Hello @bpetit,

No there is no challenge to mix the both options. However I made it "incompatible" because, the default value of -p is 5 even if you don't specify it. I thought it could be misleading, because it could truncate the filtered output silently.

Reply:
I understand. Actually it seems fine to me too to declare those options incompatible.